en/docs/tutorials/create-your-first-rest-api.md

```diff
@@ -97,97 +99,182 @@ In this tutorial, let's design the REST API by updating the low-code diagram as
+      
+       4. Double-click **`variable`** which is the default variable name to edit it. Enter `populationMillions` as the new variable name.
+      
+       5. To derive the number of one million groups in a country's population, you must divide the total population by one million. Therefore, the expression you need to enter is `(populationByCountry[0]?.value ?: 0) / 1000000`. Let's add it as follows:
+      
+           1. Click **`<add-expression>`**.
+       
+           2. Click **Expressions** and under **Arithmetic**, click **Es / En** to add the expression template for a division.
+         
+           3. Double-click the first **`<add-expression>`** (the dividend) and enter the following as the expression:
+         
+              ```
+              (populationByCountry[0]?.value ?: 0)
+              ```
+              
+           4. Double-click the second **`<add-expression>`** (the divisor) and enter `1000000` as the expression.
+         
+           5. Click **Save**.
+         
+   8. To calculate the number of COVID-19 cases in each group of one million, add another variable as follows:
+   
+       1. Click the last **+** in the low-code diagram.
+      
+       2. Click **Variable**.
+      
+       3. On the variable statement, click **`var`**, and click **decimal** in the list of suggestions that appear in the **Suggestions** tab.
+      
+       4. Double-click **`variable`**, and enter `totalCasesPerMillion` as the variable name.
+      
+       5. The expression to calculate the number of COVID-19 cases in each group of one million is `totalCases / populationMillions`. Let's add it as follows:
+      
+           1. Click **`<add-expression>`** and click **Expressions**.
+         
+           2. Under **Arithmetic**, click **Es / En**. You will get the template for a division (i.e., **`<add-explression> / <add-expression>`**).
+         
+           3. Click the first **`<add-expression>`** (the dividend) and click **Suggestions**. In the list of suggestions, click **totalCases**.
+         
+           4. Click the second **`<add-expression>`** (the divisor) and click **populationMillions** in the list of suggestions.
+         
+           5. The values for `totalCases` and `populationMillions` variables are currently in integer format. Therefore, the value of the `totalCasesPerMillion` variable derived from those variables is also in the integer format. To convert it to decimal, follow these sub-steps:
+         
+               1. Select both expressions.
+            
+               2. Click **Expressions**, and under the **Type Cast** category, click **<type>Es**.
+            
+               3. Click **Suggestions** and in the list of suggestions, click **decimal**.
+            
+           6. Click **Save**.
+            
+   9. To generate the payload, add a variable as follows:
```

Comment:
So many granular steps. :)
I wonder if we can use a different approach. Like grouping these into sections, using tables to hold the parameters, etc.

Reply:
We had that approach before. But now, depending on what you are editing, you may need to single-click or double click. Therefore, we cannot give a uniform instruction for the table.